src/components/topbar.pcss

```
@@ -13,56 +13,30 @@
   .reloaded {
     align-self: center;
     font-size: 1.5em;
-    padding-right: 1em;
+    margin-right: 2em;
+    min-width: 100px;
   }
 
   .topbarLogo .logo-svg {
     animation: unset;
     cursor: pointer;
     height: 4em;
-    margin-right: 2em;
+    margin-left: 2em;
   }
 }
 
-.react-contextmenu {
-  background-clip: padding-box;
-  background-color: #fff;
-  border: 1px solid rgba(0, 0, 0, 0.15);
-  border-radius: 0.25rem;
-  color: #373a3c;
-  font-size: 16px;
-  margin: 2px 0 0;
-  min-width: 160px;
-  opacity: 0;
-  outline: none;
-  padding: 5px 0;
-  pointer-events: none;
-  text-align: left;
-  transition: opacity 250ms ease !important;
-}
-
-.react-contextmenu.react-contextmenu--visible {
-  opacity: 1;
-  pointer-events: auto;
-  z-index: 9999;
+.more-icon {
+  margin-right: 1em;
+  width: 4em;
 }
```

```suggestion
  &:hover {
    fill: #3f9dd0;
  }
}
```
I would like to have the same hoovering effect as the clear selection button